aboutMe/apps.js

```diff
@@ -0,0 +1,55 @@
+//Q1
+const page=document.querySelector("page");
+style.fontfamily="Arial, sans-serif"
+//Q2
+const nickname = document.querySelector('#nickname');  
+const favorite = document.querySelector('#favorite'); 
+const hometown = document.querySelector('#hometown'); 
+
+nickname.innerHTML = "rayG";
+favorite.innerHTML = "playing video games"; 
+hometown.innerHTML = "Riyadh";
+
+const li=document.querySelector('li');
+for(let listitem= 
```

Comment:
I think you need to review the for-loop.
```suggestion
const li=document.querySelectorAll('li');
for(let i =0; i < li.length; i++){
  li[i].classList.add('listitem');
  li[i].style.color="rebeccapurple";
}
```